### ISSUE TITLE
Certain Mods Not Being Added

### ISSUE THREAD

**Issue by OkyDooky** [Floogen/Stardrop#110]
I tried adding [these](https://www.nexusmods.com/stardewvalley/mods/9838) [mods](https://www.nexusmods.com/stardewvalley/mods/9916) through the drop down menu (File, Add Mod, etc.), but they do not show up in the mod list and do not affect the game, either. I have the [prerequisite mod](https://www.nexusmods.com/stardewvalley/mods/5450) installed and it has been working fine. I thought maybe they weren't showing up because they are supposed to be applied through the prerequisite mod, but nothing has changed for the characters. The parent mod gives an in-game menu to enable or disable its effect on a per-character basis, but that just had them go back to their vanilla appearances.
I get no error message and I not familiar enough with the modding process to diagnose what may be going wrong here. So, I am opening this issue, because I don't know whether it's an issue with Stardrop or the mods or both.

**Comment by Floogen** [comment]
The two examples you gave lack a `manifest.json` and therefore not detected as a mod by either SMAPI or Stardrop. Those packs seem to require a manual install process of dragging and dropping a folder into another mod folder (Seasonal Outfits), which is not a supported process by Stardrop.

I'll update Stardrop to give a warning that no `manifest.json` was found and therefore no mod was installed.

**Comment by OkyDooky** [comment]
Thank you for responding. I figured that out once I opened up their archive packages and looked for myself, but forgot to report back here. The "solution" I found was to unpack both the source mod and the other mods, merging the folders and overwriting the contents, then repackaging the contents into its own unique archive and adding that as a separate mod in Stardrop.
If there's documentation re: Stardrop, then perhaps that could be added to it.
Giving a warning could definitely help. It could also give a pointer to the documentation to explain what the modder might be encountering and how to resolve it.
